### DISCUSSION TITLE
Animation problem with hidden elements

### DISCUSSION THREAD

**Discussion by Dimitri-bonnet** [swup/swup#385] [Help] [answered]
Hello, I am using swup in my project but I am having a problem with some elements that are hide for the adaptability of my site. The animations between the pages work but as soon as an element animated by swup is in display: none it no longer works. I tried with the media queries but nothing helps. How to deal with the problem?

**Comment by AugustMiller** [comment, marked as answer]
This may just be a limitation of CSS. Altering the `display` at the same time as other “transitionable” properties will always clobber them, and make the element appear for the first time with in the “final” state of any transitions.
